code/modules/mob/living/basic/vermin/mouse.dm

```diff
@@ -183,7 +195,7 @@
 
 /// Creates a new mouse based on this mouse's subtype.
 /mob/living/basic/mouse/proc/create_a_new_rat()
-	new /mob/living/basic/mouse(loc)
+	new /mob/living/basic/mouse(loc, TRUE)
```

Comment:
```suggestion
	new /mob/living/basic/mouse(loc, /* tame = */ TRUE)
```

Reply:
I was wondering how to do this without it runtiming and that works

Reply:
Changing it from an init variable to a proc also works fine (create the mouse, then call `.tame()` on it) but this isn't problematic